internal/provisioner/controller/gateway_test.go

```diff
@@ -1267,6 +1267,52 @@ func TestGatewayReconcile(t *testing.T) {
 			},
 		},
 
+		"If ContourDeployment.Spec.Contour.PodAnnotations is specified, the Contour pods' have annotations for prometheus & user-defined": {
```

Comment:
doesn't look like this test checks for the prometheus annotations as stated here?